tests/helpers/globify.ts

```diff
@@ -0,0 +1,9 @@
+// takes a string like `foo_bar`, recursively splits it up along underscores, and collects all kinds of glob expressions.
```

Comment:
```suggestion
// Takes a string like `foo_bar`, recursively splits it up along underscores, and collects all kinds of glob expressions.
```

🤓 

(Comment is super helpful, thanks!)